inception-project/inception
# #1216 - Linking span to document annotations

**What's in the PR**
- Adjust event inheritance for DocumentMetadata annotation events to match those of the other layer types
- Removed code which tried to channel feature states through the annotator state and instead rely on the feature state being passed along as part of arming a slot
- add listening for RenderSlotsEvent for QualifierFeatureEditor and SubjectObjectFeatureEditor

**How to test manually**
1. Create a new document annotation layer with a link feature to a span annotation layer.
2. Open a document
3. Create a document annotation with the type of your new layer
4. Select the new slot.
5. Annotate something in the text.
6. Link should be created

Also try out having a span layer with a link to another span.
Then have a document annotation and a span annotation both with links open and play around to see if the slots are armed/disarmed correctly.

**Automatic testing**

**Documentation**
